index.bs

```diff
@@ -168,6 +168,9 @@ interface VRDisplay : EventTarget {
    */
   void submitFrame();
 };
+
+VRDisplay implements EventTarget;
+VRDisplay implements VRDisplayEventHandlers;
```

Comment:
Again, this was picked up from WebBluetooth. Not sure what the reasoning was there. Worth asking them.